EDDiscovery/3DMap/FormMap.cs

```diff
@@ -785,9 +799,159 @@ private void RenderGalaxy()
             GL.PopMatrix();
         }
 
+        private void FillExpeditions()
+        {
+            Dictionary<string, Func<DateTime>> excursions = new Dictionary<string, Func<DateTime>>()
```

Comment:
@AnthorNet are there any plans on exposing the start/end times of expeditions in JSON from EDSM?